inject-java/src/test/java/kotlin/coroutines/Continuation.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package kotlin.coroutines;
+
+// Stub class for testing Kotlin suspend methods
+public class Continuation<T> {
```

Comment:
I would say just remove the tests in inject-java. I don't know what the legal consequences are of having a class in `kotlin.coroutines` with our license.